### DISCUSSION TITLE
Consider adding elevation for points

### DISCUSSION THREAD

**Discussion by brewin** [weather-gov/api#734] [Ideas]
`stations` and `gridpoints` both contain elevation values, but `points` does not. It could just be the grid point elevation since it already contains the other grid point data, or it could use a higher resolution source.

**Comment by dangelsaurus** [comment]
`points` returns a grid which represents a 2.5km grid area, elevation for the points endpoint would be misleading.  you would probably be better off with something like https://www.opentopodata.org/
